Ratios/CalculatorViewController.swift

```diff
@@ -8,6 +8,55 @@
 
 import UIKit
 
+fileprivate struct RatioColour {
+    static let strongBackgroundColor = UIColor(red: 0, green: 0, blue: 0, alpha: 1)
+    static let weakBackgroundColor   = UIColor(red: 1, green: 1, blue: 1, alpha: 1)
+
+    static let minRatio: Int = 12
+    static let maxRatio: Int = 20
+
+    let value: Int
+    let colour: UIColor
+    
+    private static func colorFor(ratio: Int) -> UIColor {
+        var value = ratio
+        
+        if value > maxRatio {
+            value = maxRatio
+        } else if value < minRatio {
+            value = minRatio
+        }
```

Comment:
Perhaps you want to early return here? `guard` would likely work nicely too.

```swift
guard value < maxRatio else {
    return RatioColour.strongBackgroundColor
}

guard value > minRatio else {
    return RatioColour.weakBackgroundColor
}

// everything under here is between min and max
```

Means you also remove the need for `value` variable.